src/index.js

```diff
@@ -268,6 +274,40 @@ io.on('connection', function(socket) {
     done(success(users));
   });
 
+  socket.on('cl_following', async (playerTo, done) => {
+    debug('cl_following', playerTo);
+
+    if (!user) return done(error('forbidden'));
```

Comment:
If user = 0, !user would return true (which might be undesirable). In JavaScript, it is normally considered a good coding style to have if (user == null) to check whether a variable is null or undefined. Again, it is probably that you have maintained some kind of consistency throughout the other codes (e.g. user will never be an integer), and that having (!user) is effectively the same as having (user == null). My comment is just for general cases only for your reference.

Reply:
Thank you for feedback. Indeed, user will never = 0.